### DISCUSSION TITLE
The links to the nabble mailing lists are dead

### DISCUSSION THREAD

**Discussion by JohannHansing** [foodcoops/foodsoft#896] [General]
It seems like the links to the different mailing lists found in various places in the wiki are dead.

I only randomly stumbled upon a working link via foodcoops.net/foodsoft-hosting/

It would be great if the links could be updated

**Comment by kidhab** [comment]
Thanks for this notification. I just updated the last link.

**Comment by RayOei** [comment]
Closing.
